core-java-modules/core-java-uuid/src/test/resources/logback.xml

```diff
@@ -2,8 +2,8 @@
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
-            </pattern>
+            <pattern>%d{HH:mm:ss} - %msg%n</pattern>
+            <!--<pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>-->
```

Comment:
I'm checking with the editorial team to see if we have a standard pattern that is preferred to be used, or if it is flexible. Note that this change affects every other article that may be put in this module in future.

I believe in the article itself we can truncate the log output to omit the timestamp/thread/level/class/etc and just keep the message itself, checking this point with the team too.